### PR TITLE
Update event-016.md

### PR DESCRIPTION
**Pull request by catsmith**
This workshop is now scheduled. Note it has a repeat.

### CODE REVIEW

**Comment by Chilipp** [comment]
thanks @catsmith! LGTM: https://1341-267395254-gh.circle-artifacts.com/0/SORSE/programme/workshops/event-016/index.html

@eileen-kuehn, could you please update indico and add the registration URL?

**Comment by catsmith** [comment]
just making sure this hasn't been forgotten, we could do with advertising this one soon.

**Comment by Chilipp** [comment]
yes, we need the registration pages. Do you know, when @eileen-kuehn is back again? Or can @cc-a do this on indico? Because I do not have access

**Comment by cc-a** [comment]
You just scared the hell out of me. Dates on the PR currently are for 17/19 Oct but should be Nov.

I can create events in Indico but only in the test category. Happy to create a placeholder that can be bumped by @eileen-kuehn.

**Comment by eileen-kuehn** [comment]
Hi, sorry, I don't have a computer at the moment, should be back to normal tomorrow or on Wednesday.
Creating the necessary data just now :)

**Comment by eileen-kuehn** [comment]
@cc-a, I just added you as a manager to [one of the events](https://indico.scc.kit.edu/event/2134/manage/). Could you please add the chair persons? I don't have access to them right now.

File: _events/workshops/event-016.md [hunk 1]
```diff
@@ -66,8 +66,14 @@ affiliations:
...
 category: workshops
 language: English
+time:
+    - - start: 2020-10-17T07:00:00Z
+        end: 2020-10-17T10:00:00Z
+    - - start: 2020-10-19T16:00:00Z
+        end: 2020-10-19T19:00:00Z
 prerequisites: "no prerequisite knowledge needed"
 date: 2020-08-18
+last_modified_at: 2020-10-05
```
Comment (eileen-kuehn) [inline _events/workshops/event-016.md:76]:
```suggestion
last_modified_at: 2020-10-05
registration_url:
    - https://indico.scc.kit.edu/event/2135/
    - https://indico.scc.kit.edu/event/2134/
```

**Comment by eileen-kuehn** [comment]
Ah... changing to November now....

File: _events/workshops/event-016.md [hunk 2]
```diff
@@ -66,8 +66,14 @@ affiliations:
 author: *speaker
 category: workshops
 language: English
+time:
+    - - start: 2020-10-17T07:00:00Z
+        end: 2020-10-17T10:00:00Z
+    - - start: 2020-10-19T16:00:00Z
+        end: 2020-10-19T19:00:00Z
```
Comment (eileen-kuehn) [inline _events/workshops/event-016.md:73]:
```suggestion
    - - start: 2020-11-17T07:00:00Z
        end: 2020-11-17T10:00:00Z
    - - start: 2020-11-19T16:00:00Z
        end: 2020-11-19T19:00:00Z
```

**Comment by catsmith** [comment]
Yes it should be November, sorry! Trying to figure out how to edit and failing.

**Comment by catsmith** [comment]
thanks @eileen-kuehn you got there first

File: _events/workshops/event-016.md [hunk 3]
```diff
@@ -66,8 +66,17 @@ affiliations:
...
 category: workshops
 language: English
+time:
+    - - start: 2020-11-17T07:00:00Z
+        end: 2020-11-17T10:00:00Z
+    - - start: 2020-11-19T16:00:00Z
+        end: 2020-11-19T19:00:00Z
 prerequisites: "no prerequisite knowledge needed"
 date: 2020-08-18
+last_modified_at: 2020-10-05
```
Comment (Chilipp) [inline _events/workshops/event-016.md:76]:
```suggestion
last_modified_at: 2020-10-19
```

**Comment by Chilipp** [comment]
LGTM https://1404-267395254-gh.circle-artifacts.com/0/SORSE/programme/workshops/event-016/index.html

Can I merge this then @catsmith and add the events to the calendar?

**Comment by catsmith** [comment]
Yes that looks great - I have just double checked the dates/times

**Pull request merged**